tests/stat/test_chmod.c

```diff
@@ -48,26 +48,39 @@ void cleanup() {
 void test() {
   int err;
   int lastctime;
+  int fd;
   struct stat s;
-  
+
   //
   // chmod a file
   //
-  // get the current ctime for the file
+  // get the current ctime for the file, and a file handle
+  fd = open("file", O_WRONLY);
   memset(&s, 0, sizeof s);
   stat("file", &s);
   lastctime = s.st_ctime;
   sleep(1);
 
   // do the actual chmod
-  err = chmod("file", 0200);
+  err = chmod("file", 0);
```

Comment:
How about using S_IRUSR here?  Which mean the file is not writable by anyone, but avoids using 0.